### PR TITLE
Polishingbefore opensourcing

### PR DESCRIPTION
**Pull request by pbenas**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -8,14 +8,14 @@
     name='sgmanager',
     version='1.4.1',
     packages=['sgmanager', 'sgmanager.logger', 'sgmanager.securitygroups'],
-    entry_points = {
-        'console_scripts': [ 'sgmanager = sgmanager.cli:main' ]
+    entry_points={
+        'console_scripts': ['sgmanager = sgmanager.cli:main']
     },
     license='BSD',
-    author='Filip Pytloun',
```
Comment (fpytloun) [inline setup.py]: +1 :-)

**Comment by vonsch** [comment]
Looks fine for me, ack

File: setup.py (hunk 1 above)
Comment (pbenas) [inline setup.py]: "Policy says they have a right to do anything we can't stop them from doing."

"What the hell are you talking about?" An engineer shouted at her in bewildered, furious protest. "How did you know it was Policy? Who the hell told you it was Policy?"

"The engineers with the hard white hats and clubs. The girls were crying. 'Did we do anything wrong?' they said. The men said no and pushed them away out the door with the ends of their clubs. 'Then why are you chasing us out?' the girls said. 'Policy,' the men said. All they kept saying was 'Policy, Policy. What does it mean, Policy? What is Policy?"

"Didn't they show it to you?" An engineer demanded, stamping about in anger and distress. "Didn't you even make them read it?"

"They don't have to show us Policy," the old woman answered. "The law says they don't have to."

"What law says they don't have to?"

"Policy".

Comment (dankeder) [inline setup.py]: Sir, I take my hat off to you. Good taste in humour is always appreciated. :)

Comment (fpytloun) [inline setup.py]: Good to be where common sense takes place over policies.. :-)

**Pull request merged**